docs/example-bpf.md

```diff
@@ -116,25 +134,33 @@ The most basic example, just use `sudo` to start the `go-xdp-counter` program.
 Determine the host interface to attach the BPF program to and then start the go program with:
 
 ```console
-    cd src/bpfd/examples/go-xdp-counter/
+    cd bpfd/examples/go-xdp-counter/
     sudo ./go-xdp-counter -iface <INTERNET INTERFACE NAME>
 ```
 
 or (**NOTE:** TC programs also require a direction, ingress or egress)
 
 ```console
-    cd src/bpfd/examples/go-tc-counter/
+    cd bpfd/examples/go-tc-counter/
     sudo ./go-tc-counter -direction ingress -iface <INTERNET INTERFACE NAME>
 ```
 
+or
+
+```console
+    cd bpfd/examples/go-tracepoint-counter/
+    sudo ./go-tracepoint-counter
+```
+
 The output should show the count and total bytes of packets as they pass through the
 interface as shown below:
 
 ```console
     sudo ./go-xdp-counter -iface ens3
-    2022/12/02 15:59:34 Reading /etc/bpfd/gocounter.toml ...
-    2022/12/02 15:59:34 Read /etc/bpfd/gocounter.toml failed: err open /etc/bpfd/gocounter.toml: no such file or directory
-    2022/12/02 15:59:34 Using Input: Interface=ens3 Priority=50 Source=bpf_bpfel.o
+    2022/12/02 15:59:34 Reading /etc/bpfd/bpfd.toml ...
+    2022/12/02 15:59:34 Read /etc/bpfd/bpfd.toml failed: err open /etc/bpfd/bpfd.toml: no such file or directory
```

Comment:
Do we wan't to remove printing this error? It always makes me think the examples are broken and I'm sure a naive user will feel the same

Reply:
Or rewrite the log to something like:
`2022/12/02 15:59:34 Unable to read /etc/bpfd/bpfd.toml, using default values.`
What do you think?

Reply:
Yeah that's fine too!